models/models.go

```diff
@@ -44,6 +44,7 @@ type AppModel struct {
 	Summary      string                              `json:"summary,omitempty" yaml:"summary,omitempty"`
 	Description  string                              `json:"description,omitempty" yaml:"description,omitempty"`
 	Environments []envmanModels.EnvironmentItemModel `json:"envs,omitempty" yaml:"envs,omitempty"`
+	Meta         map[string]interface{}              `json:"meta,omitempty" yaml:"meta,omitempty"`
```

Comment:
you also need to normalize the Meta field after unmarshaling the YAML model.
By default, the YAML parser unwraps a key-value pair into a map[interface]interface (even if the key is a string).
Because of this, we can not JSON marshal this kind of go model, but in some cases the cli stores the config as a json model, we solved this question when we introduced Meta field for steps, pls check this pr: https://github.com/bitrise-io/stepman/pull/269/files

Reply:
Added normalize, pls check.